chainer/function_node.py

```diff
@@ -865,7 +890,11 @@ def _backprop(outputs, inputs, grad_required, retain_grad, grads):
                 # Accumulate the duplicated gradients here
                 cur_gx = grads.get(node, None)
                 if cur_gx is not None:
-                    g = g + cur_gx
+                    if x.creator is None:
```

Comment:
Should we enclose this change with `if func.lazy_grad_sum:`?

Reply:
Yes. It reminder me that the cur_gx may be tuple, because the current function's sibling may enable lazy_grad_sum. I'd like to support `variable + tuple of variable` in `+` operator. Do you agree?

Reply:
Yes. It also reminder me that the cur_gx may be tuple, because the function's sibling may enable lazy_grad_sum. Also fixed such cases.